packages/lit-virtualizer/src/test/scenarios/testing-setup.test.ts

```diff
@@ -4,24 +4,32 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
- import { ignoreWindowErrors } from "../helpers.js";
- import { expect } from "@esm-bundle/chai";
+import { ignoreBenignErrors } from "../helpers";
```

Comment:
Nit: Enforce import extension:`../helpers.js`

Reply:
Got it!  Thanks :)